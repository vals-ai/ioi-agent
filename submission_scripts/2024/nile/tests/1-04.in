version https://git-lfs.github.com/spec/v1
oid sha256:390f7c98adf4fee46d377cff4465682931c81a56ea0b159fe7118fc913a92d36
size 43313
