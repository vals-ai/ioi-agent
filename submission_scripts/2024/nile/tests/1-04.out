version https://git-lfs.github.com/spec/v1
oid sha256:5f5d21509920f69aeaad9c84f6d8b43b5c939925a4b47d8644c8f5f7f48ddf0a
size 101
